src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,11 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+	/**
+	 * @return the feedbackToUser
+	 */
+	public String getFeedbackToUser() {
+		return feedbackToUser;
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)